packages/taikoon-ui/src/components/sections/Footer.section.svelte

```diff
@@ -183,6 +186,8 @@
     'text-content-secondary',
     'font-normal',
   );
+
+  $: hoveredIcon = 'none' as ISocialLink;
 </script>
 
 <Section height={windowSize === 'sm' ? 'fit' : 'min'} background="footer" class={sectionClasses} width="xl">
```

Comment:
lgtm but i think the footer section vh might be better suited at 55vh

Reply:
Tackling on future PR